Terasology/Apiculture
# Add apiary block with dummy UI

Add the apiary block in a non-functional state, with its UI and associated widgets.

To test:
- Give apiary
- Place and interact, and observe the UI appears containing:
     - Two vertically aligned slots, with a yellow bar to the right of the upper one
     - A honey comb of 7 slots on the right
     - The player's inventory.
